src/commands/wallet/new.ts

```diff
@@ -0,0 +1,37 @@
+import { Command, flags } from '@oclif/command';
+import { MnemonicKey } from '@terra-money/terra.js';
+import * as fs from 'fs';
+
+export default class WalletNew extends Command {
+  static description = 'Generate a new wallet.';
+
+  static flags = {
+    outfile: flags.string({
+      description: 'absolute path to store the mnemonic key to. If omitted, output to stdout',
+    }),
+  };
+
+  async run() {
+    const { flags } = this.parse(WalletNew);
+    this.log('Generating new terra-wallet');
+
+    const mk = new MnemonicKey({
+      index: 0,
```

Comment:
Shouldn't we allow also an input for the index? :thinking:

Reply:
What a great idea 👍